src/test/rpc/AccountSet_test.cpp

```diff
@@ -211,13 +211,19 @@ class AccountSet_test : public beast::unit_test::suite
 
         uint32 xfer_rate = 2000000000;
         jt[sfTransferRate.fieldName] = xfer_rate;
-        env(jt);
+        env(jt, ter(temBAD_TRANSFER_RATE));
```

Comment:
If we're setting the transfer rate to the maximum allowed, why would this fail? It seems like it should not.

Also, we should test two cases: amendment disabled (so, the _old_ behavior should be allowed) and amendment enabled (so the _new_ behavior is allowed).